lockfile.json

```diff
@@ -63,7 +63,7 @@
     "revision": "d3976b27df8622ed17bef6dd5e358b398e73c676"
   },
   "jsonc": {
-    "revision": "32cad29b7c75dbc86d793276b729d99a9017a38c"
+    "revision": "warning: redirecting to https://gitlab.com/WhyNotHugo/tree-sitter-jsonc.git/"
```

Comment:
bad bot!

Reply:
Maybe it was with `.git` there before because of this